pkg/cmd/doctor/run.go

```diff
@@ -88,9 +88,9 @@ func Run(ctx context.Context, dev *model.Dev, devPath string, c *kubernetes.Clie
 	files := []string{summaryFilename}
 	files = append(files, stignoreFilenames...)
 
-	deploymentLogs := filepath.Join(config.GetDeploymentHome(dev.Namespace, dev.Name), "okteto.log")
-	if model.FileExists(deploymentLogs) {
-		files = append(files, deploymentLogs)
+	deploymentLogsPath := filepath.Join(config.GetDeploymentHome(dev.Namespace, dev.Name), "okteto.log")
```

Comment:
With this change, will `okteto doctor` also include the main `okteto.log` file?

Reply:
From https://github.com/natefinch/lumberjack:
```
Lumberjack assumes that only one process is writing to the output files. Using the same lumberjack configuration from multiple processes on the same machine will result in improper behavior.
```
The idea is to log just to the standard output for other commands, and just for `up`, store the logs in the okteto deployment folder to make it available for `okteto doctor`. I think it is a good compromise, thoughts?

Reply:
What do you mean by "the standard output of the other commands"?

Reply:
I couldn't find any logging library that supports several processes writing to the same log file, which is what we do right now if you execute `okteto` commands in parallel.
I have checked other CLI tools and they don't usually log to a file. We log things for other commands, but I haven't found that info useful more than for troubleshooting `okteto up`.

Reply:
yeah, `okteto up` is the more complex command we have by far.  cool, if we miss the logs we can always bring them back.